woocommerce-gateway-ebanx/templates/servipag/payment-processing.php

```diff
@@ -14,5 +14,5 @@
 ?>
 
 <div class="woocommerce-message">
-	<span><?php printf( wp_kses( __( 'Payment successfully made using %1$s oxxo in %2$s.', 'woocommerce-gateway-ebanx' ), array( 'strong' => array() ) ), '' ); ?></span>
+	<span><?php _e('Payment successfully made using Servipag.', 'woocommerce-gateway-ebanx' ); ?></span>
```

Comment:
Appreciate the simplified translation call. Is it part of this pull request or a quality one? Could this pull request be also considered a quality one?